src/drivers/rc_input/RCInput.cpp

```diff
@@ -717,7 +717,7 @@ This module does the RC input parsing and auto-selecting the method. Supported m
 
 	PRINT_MODULE_USAGE_NAME("rc_input", "driver");
 	PRINT_MODULE_USAGE_COMMAND("start");
-	PRINT_MODULE_USAGE_PARAM_STRING('d', "/dev/ttyS3", "<file:dev>", "RC device", true);
+	PRINT_MODULE_USAGE_PARAM_STRING('d', "defined by RC_SERIAL_PORT", "<file:dev>", "RC device", true);
```

Comment:
Actually, not sure about this part. Maybe should we should leave the default empty (nullptr) instead?

Reply:
I am preety sure its ok:
https://github.com/PX4/PX4-Autopilot/blob/b75abf14666cd1968c1ac5af17d03c8d04677306/src/drivers/rc_input/RCInput.cpp#L117-L119

Reply:
That's true, but I was worried about the string "defined by RC_SERIAL_PORT", rather than the actual value of RC_SERIAL_PORT. There are also boards that don't have RC_SERIAL_PORT.

This seems like the right thing to do for the command line, but it might screw up the generated module documentation.

``` C++
#if defined(RC_SERIAL_PORT)
	PRINT_MODULE_USAGE_PARAM_STRING('d', RC_SERIAL_PORT, "<file:dev>", "RC device", true);
#else
	PRINT_MODULE_USAGE_PARAM_STRING('d', nullptr, "<file:dev>", "RC device", true);
#endif
```

Reply:
got you now, done here:
https://github.com/PX4/PX4-Autopilot/pull/16042/commits/84df934398783d428017e2edab980cd2cd54db91